tests/functional/testConnection.js

```diff
@@ -30,16 +30,19 @@ describe('Connectivity', function() {
     await vpn.setSetting('connectionChangeNotification', 'true');
     await vpn.clickOnQuery(queries.screenHome.CONTROLLER_TOGGLE.visible());
 
-    await vpn.waitForCondition(async () => {
-      let connectingMsg = await vpn.getQueryProperty(
-          queries.screenHome.CONTROLLER_TITLE, 'text');
-      return connectingMsg === 'Connecting…';
-    });
-
-    assert.equal(
-        await vpn.getQueryProperty(
-            queries.screenHome.CONTROLLER_SUBTITLE, 'text'),
-        'Masking connection and location');
+    // This bit is commented out because the UI has slightly changed 
+    // with the work in VPN-5312 and as a result the Connecting screen doesn't show.
```

Comment:
Is VPN-5312 the correct ticket? VPN-5312 is the ticket that tracks the change in this PR.

Reply:
Yes, it's referring to the work in the same PR. The plan is to have a front end engineer tidy up the transitions and then we can re-enable the 'connecting' bits